src/executor/Scheduler.ts

```diff
@@ -189,6 +200,22 @@ export class Scheduler {
         return new JobTracker(jobs);
     }
 
+    private createTsE2EJobTracker(): JobTracker {
+        const tsE2EPlugins: CplacePlugin[] = [];
+        this.plugins.forEach(plugin => {
+            if (plugin.hasTypeScriptE2EAssets) {
+                tsE2EPlugins.push(plugin);
+            }
+        });
+
+        const jobs: JobDetails[] = tsE2EPlugins.map(plugin => new JobDetails(
+            plugin.pluginName,
+            [],
```

Comment:
This has to be adapted :) The two parameters need to be treated same as in the other job trackers to find the dependents/dependencies with E2E TypeScript